examples/test/Test.cs

```diff
@@ -130,15 +128,78 @@ public static void Main(string[] args)
         irc.OnError += new ErrorEventHandler(OnError);
         irc.OnRawMessage += new IrcEventHandler(OnRawMessage);
 
-        string[] serverlist;
-        // the server we want to connect to, could be also a simple string
-        serverlist = new string[] {"irc.freenode.org"};
-        int port = 6667;
+        // ===============================================
+        // Standard IRC over TCP
+        // ===============================================
```

Comment:
I would prefer if you can make one project example for regular TCP and one for websockets instead of all the comments :-D